src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    public void doWork() {
+        System.out.println("Excavator started its work.");
+    }
+
+    public void stopWork() {
+        System.out.println("Excavator stopped working.");
```

Comment:
Similar to the 'doWork' method, the message in 'stopWork' should also be more informative. Make sure it indicates that it is an Excavator that has stopped working.